instrumentation/active_job/lib/opentelemetry/instrumentation/active_job/handlers/default.rb

```diff
@@ -41,9 +41,9 @@ def start(name, id, payload)
           # @return [Hash] with the span and generated context tokens
           def start_span(name, _id, payload)
             span = tracer.start_span(name, attributes: @mapper.call(payload))
-            tokens = [OpenTelemetry::Context.attach(OpenTelemetry::Trace.context_with_span(span))]
+            token = OpenTelemetry::Context.attach(OpenTelemetry::Trace.context_with_span(span))
```

Comment:
nit: since this bit of code is [repeated in the Enqueue class](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1018/files#diff-d693657995fb181905c5c85e82859e4812b1bfa4e4b3adcacba234b5a4842c00R33), does it make sense to move it to some sort of shared helper method?

Reply:
Perhaps there is an opportunity to refactor out a template method here. 🤔 let me take a stab at that in a follow up PR.

Reply:
I took a look again. I think that adding a template method here is more complex than I would like since the Enqueue class is doing things like setting a span kind, attributes, and context propagation, which the Default handler does not do. 

If I add a helper method just for the return statement, it would effectively be calling a method with two values that return a hash and that would also result in structural duplication so I am not sure how much better off we would be.